### PR TITLE
chore : support ssr

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/calendar/src/util/dom.ts [hunk 1]
```diff
@@ -112,7 +112,10 @@ export function isOverlapped(el1: Element, el2: Element) {
   return !(r1.top > r2.bottom || r1.right < r2.left || r1.bottom < r2.top || r1.left > r2.right);
 }
 
-const elProto = Element.prototype;
+// for ssr
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const ElementClass = typeof Element === 'undefined' ? function () {} : Element;
```
Comment (lja1018) [inline apps/calendar/src/util/dom.ts:117]: util에 있는 `noop` 함수를 사용해도 될 것 같습니다!

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: apps/calendar/package.json [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
-  "main": "dist/tui-calendar.js",
+  "main": "dist/toastui-calendar.js",
   "types": "types/index.d.ts",
   "sideEffects": [
     "*.css"
   ],
   "module": "dist/esm/",
-  "exports": {},
+  "exports": {
+    "./": "./"
```
Comment (jung-han) [inline apps/calendar/package.json:12]: @lja1018 @jungeun-cho 
아 그 나중에 배포해주실 때 차트처럼 js파일 뿐만아니라 css파일도 exports에 포함시켜서 내보내주셔야 snowpack사용하는 측에서 잘 사용할 수 있는 것 같더라구요ㅎㅎ참고 부탁드릴게요!

**Review by jung-han — APPROVED** [review]
완료입니다!

**Pull request merged**